File: backend/Mappers/ImageMapper.cs
using DreamBid.Dtos.Image;
using DreamBid.Models;

namespace DreamBid.Mappers
{
    public static class ImageMappers
    {
        public static ImageDto ToImageDto(this Image image, string? imageData = null)
        {
            return new ImageDto
            {
                Id = image.Id,
                FileName = image.FileName,
                Length = image.Length,
                Image = imageData
            };
        }
    }
}

File: frontend/src/mappers/ImageMapper.js
import ImageDto from "../dto/image/imageDto";

export default class ImageMapper{

    static ToImageDto(data){

        let imageDto = new ImageDto();
        imageDto.id = data.id;
        imageDto.fileName = data.fileName;
        imageDto.length = data.length;
        imageDto.image = data.image;
        return imageDto;
    }
}